core.cc

```diff
@@ -43,17 +43,16 @@ static char ident[1024];
 
 // wrap: void openlog(const char *ident, int option, int facility);
 NAN_METHOD(OpenLog) {
-    NanScope();
```

Comment:
Actually I'm not sure. This was done by the `1to2` tool. I'll double check, though.